.circleci/config.yml

```
@@ -109,95 +117,102 @@ jobs:
-          if [ -n "$CIRCLE_PR_NUMBER" ]; then
-            echo "Cannot pass creds to forked PRs, so marking this step successful"
-            circleci step halt
-          fi
-    - *build
-    - run:
-        name: Install dependencies
-        command: pip install mkdocs markdown-include
-    - add_ssh_keys:
-        fingerprints:
-          "ab:b5:f7:55:92:0a:72:c4:63:0e:57:be:cd:66:32:53"
-    - run:
-        name: Build and deploy docs
-        command: |
-          PATH="venv/bin:$PATH" script/generate_docs --output_dir=generated_docs/
-          cd generated_docs/
-          mkdocs gh-deploy -m "[ci skip] Deployed {sha} with MkDocs version: {version}"
+      - checkout
+      - &skip_forked_pr
+        run:
+          name: Early return if this build is from a forked PR
+          command: |
+            if [ -n "$CIRCLE_PR_NUMBER" ]; then
+            echo "Cannot pass creds to forked PRs,
+              so marking this step successful"
+              circleci step halt
+            fi
+      - *build
+      - run:
+          name: Install dependencies
+          command: pip install mkdocs markdown-include
+      - add_ssh_keys:
+          fingerprints: "ab:b5:f7:55:92:0a:72:c4:63:0e:57:be:cd:66:32:53"
+      - run:
+          name: Build and deploy docs
+          command: |
+           PATH="venv/bin:$PATH" script/generate_docs \
+                --output_dir=generated_docs/
+            cd generated_docs/
+           mkdocs gh-deploy -m "[ci skip] Deployed
+            {sha} with MkDocs version: {version}"
   deploy:
     parameters:
       image:
         type: string
     docker:
       # bash required for step: Determine docker image name
-    - image: relud/docker-bash
+      - image: relud/docker-bash
     steps:
-    - checkout
-    - setup_remote_docker:
-        docker_layer_caching: true
-    - run:
-        name: Determine docker image name
-        command: echo 'IMAGE="${CIRCLE_PROJECT_USERNAME+$CIRCLE_PROJECT_USERNAME/}${CIRCLE_PROJECT_REPONAME:-bigquery-etl}:${CIRCLE_TAG:-latest}"' > $BASH_ENV
-    - run:
-        name: Build docker image
-        command: docker build . --pull --tag "$IMAGE"
-    - run:
-        name: Deploy to Dockerhub
-        command: |
-          echo "${DOCKER_PASS:?}" | docker login -u "${DOCKER_USER:?}" --password-stdin
-          docker push "$IMAGE"
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Determine docker image name
+          command:
+            echo 'IMAGE="${CIRCLE_PROJECT_USERNAME+$CIRCLE_PROJECT_USERNAME/} \
+            ${CIRCLE_PROJECT_REPONAME:-bigquery-etl}:${CIRCLE_TAG:-latest}"'
+            > $BASH_ENV
+      - run:
+          name: Build docker image
+          command: docker build . --pull --tag "$IMAGE"
+      - run:
+          name: Deploy to Dockerhub
+          command: |
+           echo "${DOCKER_PASS:?}" \
+            | docker login -u "${DOCKER_USER:?}" --password-stdin \
```

Do not put a line continuation on this line, because these are suppose to be two statements.

```suggestion
            echo "${DOCKER_PASS:?}" | \
                docker login -u "${DOCKER_USER:?}" --password-stdin
```

---

This is still an issue.